src/pytest_bdd/parser.py

```diff
@@ -22,6 +22,18 @@
 STEP_PARAM_RE = re.compile(r"<(.+?)>")
 
 
+def get_tag_names(tag_data: list[GherkinTag]) -> set[str]:
```

Comment:
not important: I'd prefer if this just accepted one single `GerkinTag`, and return one string. It should be up to the caller to call it for each  GherkinTag.

Reply:
This is maintaining both previous behaviour with old parser, and what the current logic is - this has always done this - I just moved it out of the scope of the feature file parser, as it needed to be available to the examples so that the pytest-bdd parser consistently represents all tags.

I think this should be a separate PR to change this behaviour?

Reply:
Yes no worries, that's why I said optional

Reply:
Apologies - I misunderstood "not important" to mean the change I made is not important.